namerd/iface/interpreter-thrift/src/main/scala/io/buoyant/namerd/iface/ThriftInterpreterInterfaceConfig.scala

```diff
@@ -19,7 +19,7 @@ case class ThriftInterpreterInterfaceConfig(
   protected def defaultAddr = ThriftInterpreterInterfaceConfig.defaultAddr
 
   @JsonIgnore
-  def mk(interpeters: Ns => NameInterpreter, namers: Map[Path, Namer]): Servable = {
+  def mk(interpeters: Ns => NameInterpreter, namers: Map[Path, Namer], store: DtabStore): Servable = {
```

Comment:
Not your change, but it would be good to correct the spelling of "interpreters" here.